### ISSUE TITLE
(A) Radio button focus issue

### ISSUE THREAD

**Issue by otis-bath** [open-sdg/open-sdg#1757]
**High Level (A)**

When a radio button is selected, the page automatically updates. Focus has not been managed which means that keyboard only users need to press ‘tab’ to return to the radio group. Screen reader focus is taken to the beginning of the page and must navigate back down the page to find their place.
![image](https://user-images.githubusercontent.com/90769108/182149315-f5ef4bc4-9e58-44ce-9813-90c370931adf.png)
For example on: https://sdgdata.gov.uk/15-2-1/

This user comment explains the issue well: 
“I found when I was selecting the radio buttons if the focus is on a radio button, I am then able to use the cursor keys to move up or down to the next radio button. However, if I then try to use the cursor keys again to move up or down, it doesn’t work, and I must use the ‘tab’ key to get the focus back to where I want it.”

![image](https://user-images.githubusercontent.com/90769108/182149610-fdddb218-25f7-4a24-8af6-3e27da232261.png)

Effectively this is an issue of focus and control, we need to ensure that focus is taken back to the expected control so that users do not become disorientated.
